docs/design.md

```diff
@@ -128,3 +128,28 @@ The `DockerHubRegistry` requires a `user`, `pass`, and `org` field
 to be set inside the `registry` section of the configuration file. The user
 credentials are used to authenticate API queries, and the organization is the
 target org that APBs will be loaded from.
+
+### Partner RHCC Adapter
+
+The `PartnerRhccAdapter` allows the broker to be bootstrapped from the the partner registry. Like the `DockerHubRegistry` adapter, it will retrieve a list of APBs and load their specs.  The `PartnerRhccAdapterner` requires a valid RHN `user`, and `pass` for the API queries.
+
+The below is an example of how the [broker configuration file](../etc/example-config.yaml) may be configured for the `PartnerRhccAdapter`:
+
+```yaml
+---
+registry:
+  - type: partner_rhcc
+    name: reg
+    url:  https://registry.access.redhat.com
```

Comment:
shouldn't this be registry.connect.redhat.com?

Reply:
yes... I will fix